### DISCUSSION TITLE
*revives*

### DISCUSSION THREAD

**Discussion by vghfr** [Eitantan/IridiumLang#13] [Ideas]

**Comment by Eitantan** [comment]
do you want to keep working on it?

**Comment by vghfr** [comment]
Yes

**Comment by Eitantan** [comment]
ok go on repl.it tomorrow at 5:30 mst

**Comment by vghfr** [comment]
If you're available now I can hop on

**Comment by vghfr** [comment]
Before we continue, we need to organize the files in such a way that it will work with repl.it and have both versions avaliable

**Comment by vghfr** [comment]
i suggest having 2 separate repos under the iridium account
